controllers/subscription/subscription_controller.go

```diff
@@ -115,9 +118,54 @@ func (r *SubscriptionReconciler) Reconcile(ctx context.Context, request ctrl.Req
 		return ctrl.Result{}, nil
 	}
 
+	// Remove code below post 1.38.0 delivery to production
+	// Temporary code to update backend redis 100mln to specific value
+	installation := &integreatlyv1alpha1.RHMI{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rhoam",
+			Namespace: "redhat-rhoam-operator",
+		},
+	}
+	err := r.Get(context.TODO(), k8sclient.ObjectKey{Name: installation.Name, Namespace: installation.Namespace}, installation)
```

Comment:
This is fine I believe, just curious, why GetRhmiCr was not used to get RHMI CR here?

Reply:
No need, it is a tmp code and it is clear what it is going on. I just wanted to make sure I am not missing something